setup.py

```diff
@@ -29,7 +29,6 @@
     'pytest-mock',
     'pytest-cov',
     'pytest-env',
-    'pytest-sugar',
     'pytest-xdist',
```

Comment:
Can you get rid of the `-nauto` in `pytest.ini` as well? You should edit the travis files to make the tests run using `-nauto` and it shouldn't be a default option.

Like we discussed lets not run the tests in parallel by default.  I found both sugar and xdist very annoying to work with. With sugar when I tried to use the `-s` switch it wouldn't spit out the logs immediately at times.